### PR TITLE
Occurrence Code Blocks

### PR DESCRIPTION
**Pull request by pickjasmine**
Adds a button to each occurrence to show the whole JSON snippet, in case anyone needs to see more info than we are displaying or if we are not yet handling that specific occurrence kind

### CODE REVIEW

**Review by RobertKelly — APPROVED** [review]
I really like this.  And so glad it's not YAML.

File: components/occurrences/OccurrenceCodeBlock.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * limitations under the License.
+ */
+
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Button from "components/Button";
+import styles from "styles/modules/Occurrences.module.scss";
+
+const OccurrenceCodeBlock = ({ occurrence }) => {
+  const [showCode, setShowCode] = useState(false);
```
Comment (aalsabag) [inline components/occurrences/OccurrenceCodeBlock.js:23]: When did this become possible? I swear this is something new. :)

Comment (pickjasmine) [reply components/occurrences/OccurrenceCodeBlock.js:23]: Anything is possible with JavaScript ✨

**Review by aalsabag — APPROVED** [review]
lgtm

**Pull request merged**